core/schema/function.graphqls

```diff
@@ -87,7 +101,15 @@ input FunctionCallInput {
 scalar TypeDefID
 
 extend type Query {
-  typeDef(id: TypeDefID): TypeDef!
+  """
+  Create a new TypeDef with a given kind.
+  """
+  typeDef: TypeDef!
```

Comment:
Ah woops - I had an intermediate state where it took a required `kind` argument with the goal of avoiding partial instantiation, but I rolled that back; it's kind of nice to be able to skip it since it's set for you by `withListOf` and `withObject`, and it didn't fix partial instantiation anyway since for lists you need the element type, and for objects you need a name/etc.